docs/stack/about/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: About Redis Stack
+linkTitle: About
+description: Learn more about Redis Stack
+weight: 1
+---
+
+Redis Stack is an improved starting point for working with Redis. We've bundled together the best of the technology that we have to offer into an easy 
+to use package. Redis Stack extends the core capabilities of Redis OSS and provides a complete developer experience for debugging and more.
+
+In addition to all of the features of Redis OSS, Redis Stack supports:
+
+* Probabilistic data structures
+* Queryable JSON documents
+* Querying across hashes and JSON documents 
+* Time series data support (ingestion & querying)
+* Graph data models with the Cypher query language
+
+## Why Redis Stack?
+
+Redis Stack was created to allow developers to build real-time applications with a backend data platform that can reliably process requests in 
+milliseconds or less. Redis Stack does this by taking the original Redis OSS as the core and enhancing it with modern data models, data processing tools,
+and continuing to fight complexity at every turn. Ultimately, the goal of Redis Stack is to build a real-time data platform that continues to fulfill the
+philosophy of Redis OSS: simplicity, performance, and reliability. 
+
+Redis Stack unifies and simplifies the developer experience of the leading Redis modules and the capabilities they provide. Redis Stack bundles 
+five Redis modules: [RedisJSON](/docs/stack/json), [RediSearch](/docs/stack/search), [RedisGraph](/docs/stack/graph), 
+[RedisTimeSeries](/docs/stack/timeseries), and [RedisBloom](/docs/stack/bloom).
+
+The overarching vision is to provide developers with a powerful platform for all real-time data use cases. As we continue to advance the capabilities of 
+Redis beyond caching, Redis Stack is the place to start. Redis Stack delivers the core capabilities developers love about Redis and goes beyond 
+to help you build modern applications where performance is paramount. 
+
+## Redis Stack packaging
+
+There are three distinct Redis Stack packages to choose from:
+
+* **Redis Stack Server**:  This packages contains Redis OSS and module extensions only. It does *not* contain Redis Insight, the developer desktop
+application. This package is best for production deployment and is intended to be a drop-in replacement (for example, if you're already deploying 
+Redis OSS as a cache).
+
+* **Redis Stack (Desktop)**: This package contains everything a developer needs in a single bundle. This includes Redis OSS and module extensions 
```

Comment:
I think it's RedisInsight - [here](https://redis.com/redis-enterprise/redis-insight/), but I could be out of date.

Reply:
@K-Jo I had to port your changes into this PR, but it should be accurate now.